packages/common/src/lib/action/actionbar/actionbar.component.html

```diff
@@ -1,28 +1,53 @@
 <mat-list *ngIf="mode === actionbarMode.Dock">
 
-  <igo-actionbar-item
-    *ngIf="withToggleButton"
-    color="accent"
-    [withTitle]="false"
-    [withIcon]="true"
-    [color]="color"
-    [disabled]="store.view.empty"
-    [action]="toggleCollapseAction"
-    (trigger)="onTriggerAction(toggleCollapseAction)">
-  </igo-actionbar-item>
+    <div *ngIf="heightCondition && positionConditionTop && isDesktop"
+      id="topChevron">
+      <button
+        mat-icon-button
+        tooltip-position="below"
+        matTooltipShowDelay="500"
+        [matTooltip]="'igo.context.sidenav.scrollUp' | translate"
+        (click)="scrollUp()">
+        <mat-icon svgIcon="chevron-up"></mat-icon>
+      </button>
+    </div>
 
-  <ng-template *ngIf="!collapsed" ngFor let-action [ngForOf]="store.view.all$() | async">
     <igo-actionbar-item
-      [ngClass]="itemClassFunc(action)"
+      *ngIf="withToggleButton"
       color="accent"
-      [withTitle]="withTitle"
-      [withIcon]="withIcon"
+      [withTitle]="false"
+      [withIcon]="true"
       [color]="color"
-      [disabled]="store.state.get(action).disabled"
-      [action]="action"
-      (trigger)="onTriggerAction(action)">
+      [disabled]="store.view.empty"
+      [action]="toggleCollapseAction"
+      (trigger)="onTriggerAction(toggleCollapseAction)">
     </igo-actionbar-item>
-  </ng-template>
+
+    <ng-template #buttonContent *ngIf="!collapsed" ngFor let-action [ngForOf]="store.view.all$() | async">
+      <igo-actionbar-item
+        [ngClass]="itemClassFunc(action)"
+        color="accent"
+        [withTitle]="withTitle"
+        [withIcon]="withIcon"
+        [color]="color"
+        [disabled]="store.state.get(action).disabled"
+        [action]="action"
+        (trigger)="onTriggerAction(action)">
+      </igo-actionbar-item>
+    </ng-template>
+
+    <div *ngIf="heightCondition && positionConditionLow && isDesktop"
+      id="lowChevron">
+      <button
+        mat-icon-button
+        tooltip-position="below"
+        matTooltipShowDelay="500"
+        [matTooltip]="'igo.context.sidenav.scrollDown' | translate"
```

Comment:
A component in the "common" package should not use translations from "context".

Reply:
You are right. I'll work on that.